### ISSUE TITLE
Sudden problem with ammo ressuply range indicators

### ISSUE THREAD

**Issue by Lowdive** [spring1944/spring1944#377]
whenever I put my mouse over a building that would make ammo ressuply indicators show, my computer freezes a bit and the map goes yellow
![screen00000](https://user-images.githubusercontent.com/5614973/95639168-1544d180-0a5d-11eb-8b9d-bf7b5a5175ba.jpg)

I put it up as an issue because it is happening in every game now and it wasn't happening a day ago using autohost configurations (current version and engine)

[infolog.txt](https://github.com/spring1944/spring1944/files/5358023/infolog.txt)

**Comment by sanguinariojoe** [comment]
I cannot reproduce the error in my computer and I don't see nothing bad in the log (thanks by the way for submitting the issue and the attached log).

Maybe we can start checking we are not haunting the wrong monster. Can you please disable "1944 Supply Radius" (you know, pressing F11) and then check again.

Let me know

**Comment by Lowdive** [comment]
ok...
I did not change a single thing, only "event" of notice is that I loaded up the new s44 version from today (jal's balance commit for unsloped tanks) and I have supply range indicators.
![screen00001](https://user-images.githubusercontent.com/5614973/95660022-ab730900-0aea-11eb-937b-8a7a83df891f.jpg)

and no lag, most importantly.

I'm absolutely baffled, no idea how to reproduce, I will close the issue.
